build-system/tasks/helpers.js

```diff
@@ -570,6 +570,7 @@ async function minify(code, map) {
       // Settled on this count by incrementing number until there was no more
       // effect on minification quality.
       passes: 3,
+      'pure_getters': true,
```

Comment:
This is a bit safer, and will keep things known to throw an error.

```suggestion
      'pure_getters': 'strict',
```

Reply:
Thats actually the default value.
Maybe this is also resolvable within `terser`, as I'd expect something like `Object.prototype.hasOwnProperty` to be "known" not to throw.